test/client.test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const assume = require('assume');
+const sinon = require('sinon');
+const fs = require('fs');
+
+const K8Api = require('../');
+
+describe('K8Api', () => {
+  let sandbox
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+  })
+
+  afterEach(() => {
+    sandbox.restore();
+  })
+
+  describe('getInClusterConfig', () => {
+    it('should return with in-cluster config', () => {
+      // may doesn't exist (cannot stub undefined)
```

Comment:
`// You cannot stub undefined, so ensure KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT are defined`